src/gui/tray/UnifiedSearchResultItem.qml

```diff
@@ -0,0 +1,232 @@
+import QtQml 2.12
```

Comment:
for all imports please use `x.15` as we this is new code and we require Qt 5.15 so requiring `x.15` is always going to be fine